src/main/scala/no/ndla/learningpathapi/service/ImportServiceComponent.scala

```diff
@@ -36,24 +36,24 @@ trait ImportServiceComponent {
       migrationApiClient.getAllLearningPathIds match {
         case Failure(f) => Failure(f)
         case Success(liste) => Success(liste
-            .map(id => (id, doImport(id)))
-            .map { case (nid, summary) => ImportReport(nid, getStatus(summary)) })
+          .map(id => (id, doImport(id)))
+          .map { case (nid, summary) => ImportReport(nid, getStatus(summary)) })
       }
     }
 
-    def getStatus(learningPathSummaryTry: Try[LearningPathSummary]): String = {
+    def getStatus(learningPathSummaryTry: Try[LearningPathSummaryV2]): String = {
       learningPathSummaryTry match {
         case Success(_) => "OK"
         case Failure(f) => f.getMessage
       }
     }
 
-    def doImport(nodeId: String): Try[LearningPathSummary] = {
+    def doImport(nodeId: String): Try[LearningPathSummaryV2] = {
       for {
         metaData <- migrationApiClient.getLearningPath(nodeId)
         converted <- Try(upload(metaData))
         indexed <- searchIndexService.indexDocument(converted)
-      } yield converterService.asApiLearningpathSummary(converted)
+      } yield converterService.asApiLearningpathSummaryV2(converted, Language.AllLanguages).get
```

Comment:
Selv om det i teoriener trygt å gjøre `.get` når language = `Language.AllLanguages` tror jeg det er best å unngå å gjøre det. Enten putt `converterService.asApiLearningpathSummaryV2...`  i for-comprehensionen (og evt. fåt den til å returnere en `Try`), eller gjør en match etterå